testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -28,7 +28,7 @@
     @Option(names = "--server-package", paramLabel = "<package>", description = "Path to Trino server package " + DEFAULT_VALUE, defaultValue = "${server.module}/target/${server.name}-${project.version}.tar.gz")
     public File serverPackage;
 
-    @Option(names = "--without-presto", description = "Do not start " + COORDINATOR)
+    @Option(names = "--without-trino", description = "Do not start " + COORDINATOR)
     public boolean withoutPrestoMaster;
```

Comment:
please update the field, maybe `withoutCoordinator`?

Reply:
Covered by https://github.com/trinodb/trino/pull/6516 .